addons/l10n_co/data/template/account.tax-co.csv

```diff
@@ -1,225 +1,225 @@
-"l10n_co_tax_covered_goods","1","0% C A","0.0","percent","sale","tax_group_covered_goods","","","base","invoice","","Covered Assets","0%","Bienes Cubiertos"
-"","","","","","","","","","tax","invoice","","","",""
-"","","","","","","","","","base","refund","","","",""
-"","","","","","","","","","tax","refund","","","",""
+"id","sequence","name","amount","amount_type","type_tax_use","tax_group_id","active","children_tax_ids","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id","description","invoice_label",name@es,description@es
+"l10n_co_tax_1","0","19% VAT","19.0","percent","purchase","tax_group_iva_19","","","base","invoice","","VAT Purchase 19%","19%","19% IVA","IVA Compra 19%"
+"","","","","","","","","","tax","invoice","co_puc_240810","","","",""
+"","","","","","","","","","base","refund","","","","",""
+"","","","","","","","","","tax","refund","co_puc_240810","","","",""
+"l10n_co_tax_0","1","5% VAT","5.0","percent","purchase","tax_group_iva_5","","","base","invoice","","VAT Purchase 5%","5%","5% IVA","IVA Compra 5%"
+"","","","","","","","","","tax","invoice","co_puc_240810","","","",""
+"","","","","","","","","","base","refund","","","","",""
+"","","","","","","","","","tax","refund","co_puc_240810","","","",""
+"l10n_co_tax_2","1","16% D","16.0","percent","purchase","tax_group_iva_16","","","base","invoice","","Deductible VAT Purchase 16% (2016 and January 2017)","16%","16% Desc IVA","IVA Descontable Compra 16% (2016 y Enero 2017)"
+"","","","","","","","","","tax","invoice","co_puc_240810","","","",""
+"","","","","","","","","","base","refund","","","","",""
+"","","","","","","","","","tax","refund","co_puc_240810","","","",""
+"l10n_co_tax_5","1","0% IMP VAT","0.0","percent","purchase","tax_group_iva_0","","","base","invoice","","VAT Purchase Imports","0%","0% IVA Impo","IVA Compra Importaciones"
+"","","","","","","","","","tax","invoice","","","","",""
+"","","","","","","","","","base","refund","","","","",""
+"","","","","","","","","","tax","refund","","","","",""
+"l10n_co_tax_6","1","0% EXC VAT","0.0","percent","purchase","tax_group_iva_0","","","base","invoice","","VAT Purchase Excluded","0%","0% IVA Exc","IVA Compra Excluido"
+"","","","","","","","","","tax","invoice","","","","",""
+"","","","","","","","","","base","refund","","","","",""
+"","","","","","","","","","tax","refund","","","","",""
+"l10n_co_tax_7","1","0% EXEMPT","0.0","percent","purchase","tax_group_iva_0","","","base","invoice","","VAT Exempt Purchase","0%","0% IVA Exen","IVA Compra Exento"
+"","","","","","","","","","tax","invoice","","","","",""
+"","","","","","","","","","base","refund","","","","",""
+"","","","","","","","","","tax","refund","","","","",""
+"l10n_co_tax_12","1","15%  RteIVA 19%","-2.85","percent","purchase","tax_group_r_iva_285","","","base","invoice","","RteIVA 15% over 19% VAT (-2.85%)","-2.85%","15%  RteIVA 19%","RteIVA 15% sobre el 19% IVA"
```

Comment:
@ren-odoo @chklop 

What is the logic we want to apply here?  We added VAT to the English above, but when it is Rent on VAT we change back to RteIVA?

Reply:
Hi @jco-odoo we can change it ro RteVAT to be more consistent, no problem the most importat for the end user is actually the spanish translations.   @aliyatastemirova I add the updated on the file https://docs.google.com/spreadsheets/d/1JRHBCypzzHewnMeNr8MWSlftihOEhGTqNF78dwIQKW8/edit#gid=1039337638, if you prefer to do the changes on the csv directly, I replaced RteIVA by RteVAT just in the Name column, name@es remains the same. 

Thank you in advance